meld/system/builders/build_elastic_network_restraints.py

```diff
@@ -0,0 +1,150 @@
+#
+# Copyright 2015 by Justin MacCallum, Alberto Perez, Ken Dill
+# All rights reserved
+#
+
+"""
+Methods to read template coordinates from a system object and build elastic network restraints prior to simulation for a specific chain.
+"""
+
+import numpy as np  # type: ignore
+from scipy.spatial.distance import pdist  # type: ignore
+from scipy.spatial.distance import squareform  # type: ignore
+from typing import List, Optional, Union
+
+from meld.system import restraints
+from meld.system import meld_system
+from meld.system import scalers
+from meld import unit as u
+
+def create_elastic_network_restraints(
+    system: meld_system.System,
+    scaler: scalers.RestraintScaler,
+    chain_index: int,
+    cutoff: float = 1.0,
```

Comment:
Can we make this a Quantity, and add checks to make sure that units are included? If you look at the code for some of the restraints, there should be examples of how to do this.